src/compas/robots/__init__.py

```diff
@@ -115,6 +116,10 @@
 """
 from __future__ import absolute_import
 
+from .configuration import (
+    Configuration,
+    FixedLengthList
```

Comment:
Should we expose this class or rather keep it an implementation detail?

Reply:
Yeah, this is tricky... I use it in `JointTrajectoryPoint`.

Reply:
Should I make it available as a third level import rather than second?

Reply:
yes, that sounds like a good alternative